manager/volume.go

```diff
@@ -193,6 +193,7 @@ func (m *VolumeManager) Create(name string, spec *types.VolumeSpec) (v *longhorn
 			FromBackup:          spec.FromBackup,
 			NumberOfReplicas:    spec.NumberOfReplicas,
 			StaleReplicaTimeout: spec.StaleReplicaTimeout,
+			BaseImage:           spec.BaseImage,
```

Comment:
Check the node.Status.MountPropagation. If it's not supported on all the nodes, reject the base image option.